packages/jest-worker/README.md

```diff
@@ -59,6 +59,10 @@ List of method names that can be called on the child processes from the parent p
 
 Amount of workers to spwan. Defaults to the number of CPUs minus 1.
 
+#### `maxRetries: number` (optional)
+
+Maximum amount of times that a dead child can be re-spawned, per call. Defaults to `5`, pass `Infinity` to allow endless retries.
```

Comment:
Let's do 3 as a default. If something crashes the process three times in a row, it's probably broken.